adapter/i_o.py

```diff
@@ -511,7 +511,15 @@ def get_tables(
                 dict_of_dfs = Excel(file_path, pre_existing_keys).load(
                     data_object_names=table_names_to_load
                 )
-
+                
+                # if no tables were found, try reading
+                # all sheets as individual tables
+                if not dict_of_dfs:
+                    dict_of_dfs = pd.read_excel(
```

Comment:
This looks fine. Would it throw a reasonable error or should we introduce a custom error message along the lines "Please provide input data in a readable format. The readable formats are ..."?

Reply:
Hmm.  That's a good question.  I think any errors would happen downstream.  For instance, the `pd.read_excel()` call will read in something off each sheet (even for an empty sheet) without throwing any exceptions.  Similarly, I imagine that the legacy call to i_o for a workbook with no named ranges/tables would just create an empty dict?  Then, downstream issues would occur.  That said, I think I would lean toward leaving as-is (without trying to catch exceptions).

Reply:
I agree with you and I thought of something else - we have small test input files covering all compatible formats. We should add a test input file with two tabs and example data on each, just so the users have a way to understand and use the functionality.

Reply:
That makes sense.  There are a bunch of test files in the repo now.  How would you like me to name this one?  Perhaps `test_no_named_ranges.xlsx`?

Reply:
This is also something @0xd5dc may be able to easily add similarly to other methods in the `test_i_o.py` if he had the small example input file prepared.

Reply:
Sure, that's a good name.

Reply:
OK.  I just pushed up that test input file.  It just has two dummy sheets, with no named ranges; that should work with the change made in this PR.